src/networkapi.jl

```diff
@@ -1624,7 +1624,7 @@ function validate(rs::ReactionSystem, info::String = "")
             rxunits *= get_unit(sub)^rx.substoich[i]
         end
 
-        if rxunits != rateunits
+        if !isequal(rxunits, rateunits)
```

Comment:
Caused error with MTK v9, this seemed to fix it. Guess that either of these could suddenly be `Num`s or stuff like that.

Reply:
yes, I have a follow up PR for actually dealing with units, this is just the minimal amount of change for tests to pass.